### PR TITLE
Add channel export v0.2.2

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary
Adding channel export v0.2.2

#### Ticket Link
https://community-daily.mattermost.com/core/pl/318z1aiiqtyuugsam6xkdx5ckh

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2499,6 +2499,62 @@
     },
     "updated_at": "2019-06-12T19:04:44Z"
   },
+  {
+    "homepage_url": "",
```
Comment (lieut-data) [inline plugins.json:2503]: Can you file a parallel ticket to update our metadata in the repository's `plugin.json`, as well as a ticket to get an icon designed?

**Review by lieut-data — APPROVED** [review]

File: plugins.json (hunk 1 above)
Comment (crspeller) [reply plugins.json:2503]: https://mattermost.atlassian.net/browse/MM-30960
https://mattermost.atlassian.net/browse/MM-30961

File: plugins.json [hunk 2]
```diff
@@ -2499,6 +2499,62 @@
...
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-channel-export-v0.2.2.tar.gz",
+    "release_notes_url": "",
+    "labels": [
+      {
+        "name": "Enterprise (E20)",
+        "description": "This plugin only works on servers with an E20 license.",
+        "url": "https://mattermost.com/pricing/",
+        "color": ""
+      }
+    ],
```
Comment (hanzei) [inline plugins.json:2514]: @itao @aaronrothschild Should this plugin be marked as `Beta` given that it's "only" `v0.2.2`?

**Review by hanzei — CHANGES_REQUESTED** [review]
@crspeller This PR should go into `production` not `master`, correct?

File: plugins.json (hunk 2 above)
Comment (lieut-data) [reply plugins.json:2514]: We could cut a `v1.0.0` if needed, but it would just be for optics -- the feature set stabilized months ago for this plugin.

**Review by hanzei — DISMISSED** [review]
Look good code wise. I'm deferring the `Beta` question to @aaronrothschild and @itao

File: plugins.json (hunk 2 above)
Comment (itao) [reply plugins.json:2514]: @lieut-data Let's cut v1.0, the existing features are stable and wholly solves a customer need.

**Review by itao — DISMISSED** [review]
Only feedback is to denote this version as `v1.0.0`, feel free to merge after that change.

**Comment by crspeller** [comment]
Closing. We will open a new PR for 1.0.0 when the plugin build pipeline is fixed.

**Comment by lieut-data** [comment]
Going to re-open and merge v0.2.2 to start the ball rolling here while we sort out the v1.0.0 issues. There are customers actively waiting to install this, and the version bump is just for optics at this point.

**Review by lieut-data — APPROVED** [review]

**Comment by lieut-data** [comment]
@prapti, I wasn't sure how we review marketplace changes -- is that you, or would @DHaussermann normally be involved? Just wanting to verify that v0.2.2 of the channel export plugin shows up. (Note that we're missing some "metadata" like homepage_url and such, but the server should handle all that correctly.)

**Comment by prapti** [comment]
@lieut-data I have done some marketplace reviews before, and can definitely review this one once Tao and Ben are done reviewing.

**Comment by prapti** [comment]
@crspeller @lieut-data I'm testing this now but there's no Channel Export in the marketplace when I'm running this marketplace locally (I'm on the correct pr branch).
Screen Shot 2020-12-04 at 11 33 13 AM
Screen Shot 2020-12-04 at 11 33 21 AM

**Comment by lieut-data** [comment]
@prapti, can you confirm you have an enterprise license installed locally? I believe we filter out enterprise plugins (including incident management, which would also be "missing" in this context.)

**Comment by lieut-data** [comment]
(I've also pushed another update to align with the latest `production` changes.)

**Review by prapti — APPROVED** [review]
@lieut-data I did not have an EE license :) definitely been a while since I've reviewed marketplace changes.
The Channel Export plugin v0.2.2 lists as expected with an E20 license.
Approving since I realized the only pending review label on the PR is QA. If there are further changes or reviews, please let me know, I can review again!

**Review by lieut-data — APPROVED** [review]

**Pull request merged**